ncm-ssh/src/main/pan/components/ssh/schema-7.4.pan

```diff
@@ -0,0 +1,12 @@
+# #
+# Software subject to following license(s):
+#   Apache 2 License (http://www.opensource.org/licenses/apache2.0)
+#   Copyright (c) Responsible Organization
+#
+
+declaration template components/ssh/schema-7.4;
+
+type ssh_authkeyscommand_options_type = {
+            "AuthorizedKeysCommand" ? string
+            "AuthorizedKeysCommandUser" ? string
```

Comment:
the issue mentions ` sshd fails to start/ncm rolls back as it needs AuthorizedKeysCommandUser to also be set` so this should be reflected in the schema somehow.